pkg/vm/engine/disttae/types.go

```diff
@@ -239,10 +243,14 @@ func (txn *Transaction) PutCnBlockDeletes(blockId *types.Blockid, offsets []int6
 
 func (txn *Transaction) StartStatement() {
 	if txn.startStatementCalled {
-		logutil.Fatal("BUG: StartStatement called twice")
+		currentStack := string(debug.Stack())
+		logutil.Fatal("BUG: StartStatement called twice",
+			zap.String("currentStack", currentStack),
+			zap.String("lastCallback", string(txn.startStatementLastCallStack)))
 	}
 	txn.startStatementCalled = true
 	txn.incrStatementCalled = false
+	txn.startStatementLastCallStack = debug.Stack()
```

Comment:
How expensive is this Stack() thing?    Note that this is not only called when bug happens, it is called always.

Fine if we just need to catch the bug and disable.    But if we have real customer/deployment, this could be an issue.

Reply:
这个是1.0-dev上的。main上应该不能加